src/middlewared/middlewared/plugins/interface/bridge.py

```diff
@@ -8,7 +8,28 @@ class InterfaceService(Service):
     class Config:
         namespace_alias = 'interfaces'
 
-    def bridge_setup(self, bridge, sync_interface_opts):
+    @private
+    def pre_bridge_setup(self, bridge, sync_interface_opts):
+        name = bridge['interface']['int_interface']
+        members_database = set(bridge['members'])
+
+        try:
+            iface = netif.get_interface(name)
+        except KeyError:
+            return
+        finally:
+            for member in members_database:
```

Comment:
Why is it in `finally` block? It'll have more sense to be before `iface = netif.get_interface(name)`. Right now we have the code that is executed in non-obvious sequence.